docs/HowTo/Configure/Consensus-Protocols/Raft.md

```diff
@@ -4,47 +4,25 @@ description: Configuring Raft consensus
 
 # Configuring Raft consensus
 
+GoQuorum implements the Raft Proof-of-Authority (PoA) consensus protocol.
 To enable Raft consensus, specify the [`--raft`](../../../Reference/CLI-Syntax.md#raft) command line option when starting GoQuorum.
+You can [create a private network using Raft](../../../Tutorials/Private-Network/Create-a-Raft-network.md).
 
-Raft requires that all initial nodes in the cluster are configured as [static peers](https://github.com/ethereum/go-ethereum/wiki/Connecting-to-the-network#static-nodes).
+Raft requires that all initial nodes in the cluster are configured as
+[static peers](https://github.com/ethereum/go-ethereum/wiki/Connecting-to-the-network#static-nodes).
 The order of the enode IDs in the `static-nodes.json` file must be the same across all peers.
 
 The enode IDs must include a `raftport` querystring parameter specifying the Raft port for each peer.
 
 !!! example
+
     ```bash
-    `enode://abcd@127.0.0.1:30400?raftport=50400`
+    enode://abcd@127.0.0.1:30400?raftport=50400
     ```
 
 For the Raft network to work, 51% of the peers must be up and running.
 We recommend having an odd number of at least 3 peers in the network.
 
-## Adding Raft members
```

Comment:
is this information duplicated somewhere else?

Reply:
Yes, in HowTo/Use/adding_nodes.md